extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -109,8 +96,9 @@ public Job newJob(TriggerFiredBundle bundle, org.quartz.Scheduler scheduler) thr
                     for (Scheduled scheduled : method.getSchedules()) {
                         String name = triggerNameSequence.getAndIncrement() + "_" + method.getInvokerClassName();
                         JobBuilder jobBuilder = JobBuilder.newJob(InvokerJob.class)
-                                .withIdentity(name, Scheduler.class.getName()).usingJobData(INVOKER_KEY,
-                                        method.getInvokerClassName());
+                                .withIdentity(name, Scheduler.class.getName())
+                                .usingJobData(INVOKER_KEY, method.getInvokerClassName())
+                                .requestRecovery();
```

Comment:
Yes, it was previously `false`. I was thinking of marking all jobs recoverable in case of failure (hard shutdown of a node ) making the job rediscover-able by other nodes for execution.